src/Store.js

```diff
@@ -51,6 +51,26 @@ class Store {
     }
   }
 
+  /**
+   * builds an instance of a model but does not add it to the store
```

Comment:
Very nitpicky, but since the main difference between this and just using `createModel` is that it also adds a temporary ID that can either be automatically created or manually added, could you indicate that in the description here? Just like:
```suggestion
   * Builds an instance of a model that includes either an automatically or manually created temporary ID, but does not add it to the store. 
```